models/lung_nodule_ct_detection/docs/README.md

```diff
@@ -1,16 +1,15 @@
-# Description
-A pre-trained model for volumetric (3D) detection of the lung lesion from CT image.
-
 # Model Overview
-This model is trained on LUNA16 dataset (https://luna16.grand-challenge.org/Home/), using the RetinaNet (Lin, Tsung-Yi, et al. "Focal loss for dense object detection." ICCV 2017. https://arxiv.org/abs/1708.02002).
+A pre-trained model for volumetric (3D) detection of the lung nodule from CT image.
 
-LUNA16 is a public dataset of CT lung nodule detection. Using raw CT scans, the goal is to identify locations of possible nodules, and to assign a probability for being a nodule to each location.
+This model is trained on LUNA16 dataset (https://luna16.grand-challenge.org/Home/), using the RetinaNet (Lin, Tsung-Yi, et al. "Focal loss for dense object detection." ICCV 2017. https://arxiv.org/abs/1708.02002).
 
-Disclaimer: We are not the host of the data. Please make sure to read the requirements and usage policies of the data and give credit to the authors of the dataset!
+<p align="center">
```

Comment:
Hi @Can-Zhao , the way to add the figure is wrong (see: https://github.com/Project-MONAI/model-zoo/blob/de537062d7ebba3bd66b266221d7b6f695a7266e/models/lung_nodule_ct_detection/docs/README.md)

Please refer to other bundles and fix it, thanks!